library/minishift_testlib.py

```diff
@@ -6,7 +6,7 @@
 
 log = logging.getLogger("Openshift.Debug")
 
-def wait_for_output(cmd, timeout = 60):
+def wait_for_output(cmd, timeout = 180):
```

Comment:
If the function does not execute within 1 min as it was before then its better to fail. We can mark it as performance issue. WDYT ?